dashboard/dashboard.py

```diff
@@ -468,7 +468,12 @@ def get_current_datetime() -> datetime:
     selected_body = st.selectbox("Select body", bodies)
     col1, col2 = st.columns(2)
     with col1:
-       ...
+        st.header(f'{selected_body} Visibility Across The UK')
```

Comment:
Nice to have Caleb's visibility map integrated into the dashboard, important to have it next to the the aurora visibility map.